i18n/extra/de.json

```diff
@@ -198,4 +201,4 @@
             "Sun"
```

Comment:
@mwjames Must [these](https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/d35763142ec4455e07c817e5f2a8cb25f4d8632c/i18n/extra/de.json#L174-L201) be 3 chars? In German we have only 2 chars shortcuts so ...